torchgen/aoti/fallback_ops.py

```diff
@@ -16,6 +16,8 @@
     "aten.adaptive_max_pool2d.default",
     "aten.adaptive_max_pool3d.default",
     "aten.adaptive_max_pool3d_backward.default",
+    "aten.add.Scalar",
+    "aten.add.Tensor",
```

Comment:
Do we have to add this as a fallback? `aten.add.Tensor` should be a simple pointwise op and properly codegen-ed by Inductor.

Reply:
@desertfire I agree, and I'm unsure why we ran into this, but the complex fallback test specifically required this op. I'd also note that we already codegen for `mul.Scalar` and `mul.Tensor` below...

Reply:
@desertfire It appears that the test in question (`test_complex_fallback`) is deliberately triggering a situation where inductor will choose to fallback to `aten.add` (adding complex numbers). We can continue to skip this test indefinitely if you'd prefer not to add new fallback ops, but passing the test will require this fallback.

I know that there's some work ongoing with better supporting complex numbers in torch, but I don't really know the status of that, so this may be needed for a while.